daemon/mgr/container.go

```diff
@@ -367,6 +367,13 @@ func (mgr *ContainerManager) Create(ctx context.Context, name string, config *ty
 		SnapshotID: snapID,
 	}
 
+	if config.ReadonlyPaths != nil {
+		container.ReadonlyPaths = config.ReadonlyPaths
+	}
+	if config.MaskedPaths != nil {
+		container.MaskedPaths = config.MaskedPaths
+	}
```

Comment:
Same as issue mentioned above.

Reply:
unnecessary check here

Reply:
@Starnop please update the code here. Thanks